io.github.yairm210.unciv.yaml

```diff
@@ -0,0 +1,49 @@
+app-id: io.github.yairm210.unciv
+default-branch: stable
+runtime: org.freedesktop.Platform
+runtime-version: '19.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions: [org.freedesktop.Sdk.Extension.openjdk8]
+command: unciv-wrapper
+finish-args:
+  - --socket=x11
+  - --device=dri
+  - --socket=pulseaudio
+  - --share=network
+  - --env=PATH=/app/bin:/app/jre/bin:/usr/bin
+  - --env=JAVA_HOME=/app/jre
+modules:
+  - name: openjdk
+    buildsystem: simple
+    build-commands: [/usr/lib/sdk/openjdk8/install.sh]
+
+  - name: xrandr
+    sources:
+      - type: archive
+        url: https://xorg.freedesktop.org/archive/individual/app/xrandr-1.5.1.tar.xz
+        sha256: 7bc76daf9d72f8aff885efad04ce06b90488a1a169d118dea8a2b661832e8762
+
+  - name: unciv
+    buildsystem: simple
+    build-commands:
+      - install -D unciv-wrapper.sh /app/bin/unciv-wrapper
+      - install -D Unciv.jar /app/data/Unciv.jar
+      - install -Dm644 -t /app/share/metainfo io.github.yairm210.unciv.appdata.xml
+      - install -Dm644 -t /app/share/applications io.github.yairm210.unciv.desktop
+      - install -Dm644 -t /app/share/icons/hicolor/512x512/apps io.github.yairm210.unciv.png
+    sources:
+      - type: file
+        path: unciv-wrapper.sh
+      - type: file
+        path: io.github.yairm210.unciv.desktop
+      - type: file
+        path: io.github.yairm210.unciv.png
+      - type: file
+        path: io.github.yairm210.unciv.appdata.xml
```

Comment:
New appdata files should use metainfo instead. https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html

Reply:
The appsteam validator should have told me that, or it should have been written in the flathub documentation :confused: 

Regardless, thank you for your review.

> Screenshots should ideally be in the PNG format, but JPEG and WebP images are also fine. Keep in mind though that the images are converted into PNG in any case by the distributor of a software collection. 

Looks like the appstream validator rejected my webp screenshot, BTW.

Please tell me if the new version conforms better.